README.md

```diff
@@ -1,3 +1,7 @@
+# CAUTIONS!
+Current released Scalafix supports ExplicitNonNullaryApply [ONLY in Scala 2.12](https://github.com/scala/scala-rewrites/issues/31). 
+Scalafix will support 2.13 and 2.11 with cross-building ExplicitNonNullaryApply rule against them.  
```

Comment:
As someone who doesn't quite understand the mechanics of these libraries properly, allow me to attempt to clarify.

*"Scalafix v1 will allow the ExplicitNonNullaryApply rule to work and fix up code in Scala 2.11 and 2.13"*

Is that correct?

Reply:
Thank you for clarification.

> Is that correct?

It looks good to me except Scalafix version, depending on Scalafix release plans.

I'm not sure it will be effective only after Scalafix v1, since the PullRequest has been already merged and there might be additional v0.9.X releases.

So I'm fine with rewriting the sentence with:

"Scalafix will allow the ExplicitNonNullaryApply rule to work and fix up code in Scala 2.11 and 2.13"

Reply:
Of note, even coming Scalafix releases, ExplicitNonNullaryApply rule has to be cross-built by #31 PR.

Reply:
although it won't be automatic, the upcoming 0.9.18 release of sbt-scalafix (probably out tomorrow) will support running rules compiled in 2.11 and 2.13, see https://github.com/scalacenter/scalafix/pull/1166